org.eclipse.xtext.ui.tests/xtend-gen/org/eclipse/xtext/ui/tests/xmleditor/XmlContentAssistLexerTests.java

```diff
@@ -34,22 +35,26 @@
   
   @Override
   public void test022() {
-    StringConcatenation _builder = new StringConcatenation();
-    _builder.append("<a b=\"/>");
-    _builder.newLine();
-    StringConcatenation _builder_1 = new StringConcatenation();
-    _builder_1.append("RULE_TAG_START_OPEN \'<\'");
-    _builder_1.newLine();
-    _builder_1.append("RULE_ID \'a\'");
-    _builder_1.newLine();
-    _builder_1.append("RULE_WS \' \'");
-    _builder_1.newLine();
-    _builder_1.append("RULE_ID \'b\'");
-    _builder_1.newLine();
-    _builder_1.append("RULE_ATTR_EQ \'=\'");
-    _builder_1.newLine();
-    _builder_1.append("RULE_TAG_EMPTY_CLOSE \'/>\'");
-    _builder_1.newLine();
-    this.assertLexing(_builder, _builder_1);
+    try {
```

Comment:
I wonder if the try-catch block is needed here.

Reply:
generated code